submissions/DaniaB24/html-css-popup/index.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Web Site</title>
+    <link rel="stylesheet" href="style.css" type="text/css">
+</head>
+
+<body>
+    <nav class="nav-menu">
+        <ul class="nav__list">
+            <li class="nav__item"><a href="#" class="nav__link">Mail</a></li>
+            <li class="nav__item"><a href="#" class="nav__link">Images</a></li>
+            <li class="nav__item">
+                <div class="b-menu">
```

Comment:
could you please use only one style for class names? (use the only dash or only underscore for all class names)

Reply:
yeap)

Reply:
Thanks for reviewing)